packages/augur-state/src/db/test.ts

```diff
@@ -0,0 +1,151 @@
+import { EthersProvider } from "@augurproject/ethers-provider";
+import { ContractDependenciesEthers } from "contract-dependencies-ethers";
+import { Augur } from "@augurproject/api";
+import { uploadBlockNumbers } from "@augurproject/artifacts";
+import { DeployerConfiguration, ContractDeployer } from "@augurproject/core";
+import * as path from 'path';
+import * as ganache from "ganache-core";
+import { CompilerConfiguration} from "@augurproject/core/source/libraries/CompilerConfiguration";
+import { ContractCompiler } from "@augurproject/core/source/libraries/ContractCompiler";
+import { EthersFastSubmitWallet } from "@augurproject/core/source/libraries/EthersFastSubmitWallet";
+import { ContractAddresses, NetworkId } from "@augurproject/artifacts";
+import { UserSpecificEvent } from "./DB";
+
+
+// TODO Get these from GenericContractInterfaces (and do not include any that are unneeded)
+export const genericEventNames: Array<string> = [
```

Comment:
I moved this and userSpecificEvents to Augur.ts in https://github.com/AugurProject/augur/pull/1296. I also changed most of the userSpecificEvents to be genericEventNames after Paul explained that we'll actually want most of them to be considered generic. I think I should probably merge https://github.com/AugurProject/augur/pull/1296 first, and then you can update this file to import those from Augur.ts.

Reply:
KK